### PR TITLE
Create hip 0074

### PR DESCRIPTION
**Pull request by vincenzospaghetti**
Creating HIP 74 on behalf of author

### CODE REVIEW

File: 0074 [hunk 1]
```diff
@@ -0,0 +1,164 @@
+# HIP Template (Give it a title here but do not allocate a number, maintainer will allocate a number)
```
Comment (vincenzospaghetti) [inline 0074:1]: HIP 74:

Comment (vincenzospaghetti) [reply 0074:1]: @meowshka what is the HIP title?

File: 0074 [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+
+# Detailed Explanation
+
+This HIP replaces the current algorithm for MOBILE rewards that was initially introduced during the Genesis phase and was based on Radio Type multipliers with a new algorithm that uses the location of the radio to calculate rewards based on the hexes the radio actually covers, as modeled by the obstruction data oracle. Below is a detailed explanation of the proposed rewards algorithm and new requirements for scaling MOBILE rewards based on the Modeled Coverage data.
+
+## Reward Mechanics
+
+### Hex Size
+
```
Comment (vincenzospaghetti) [inline 0074:51]: I would appreciate some more background here...how are we using Hexs today (just a sentence of 2) before we go into the change.

Comment (vincenzospaghetti) [reply 0074:51]: @meowshka

File: 0074 [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+Radio 1 (Outdoor) | = 246.91x1x1x34 = 8,395
+Radio 2 (Outdoor) | = 246.91x1x0.25x12 = 741
+Radio 3 (Indoor) | = 246.91x1x0.5x7 = 864
+
+
+## Data Visualization
+
+All members of the MOBILE SubDAO need to see real-life coverage provided by 5G Radios. Modeled Coverage data will play a key role in providing information to visualize it. More data will be added as additional external sources like Mobile Mappers come into play.
+
+This HIP proposes the creation of a new Explorer dedicated exclusively to MOBILE Network. As we grow with more features and data for rewards, combining it with information about the IoT Network becomes practically impossible.
```
Comment (vincenzospaghetti) [inline 0074:139]: Is this going to use the same UI as the explorer today? Like as another page or layover? or will this be totally separate?

File: 0074 [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+![Image 2. Baicells Outdoor Radio 430 installed on the two-story building roof on the pole.](<>)
+Image 2. Baicells Outdoor Radio 430 installed on the two-story building roof on the pole.
+
+![Image 3. Modeled coverage of the Radio pictured on Image 1.](<>)
+Image 3. Modeled coverage of the Radio pictured on Image 1.
+
+
+## Transition Plan
+
+To make sure the transition to the new PoC system, which replaces static, power-based multipliers, is smooth, Hotspot owners should be able to evaluate the coverage they provide before the switch takes effect.
```
Comment (vincenzospaghetti) [inline 0074:152]: How can someone evaluate this? Can you elaborate this here?

File: 0074 [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+We propose to show Mapped Coverage information for at least four weeks before switching to the new PoC model.
```
Comment (vincenzospaghetti) [inline 0074:153]: Where? How do we access this?

File: 0074 [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+Rough timeline for the Modeled Coverage rollout:
+
+Week 0:
+<ul><li>Start displaying Modeled Coverage data as a basic map overlay of 5G data coverage with signal strength in the Mobile Explorer.</li></ul>
+Week 3:
+<ul><li>Add the ability to check coverage for each Outdoor Radio on the Mobile Explorer map.</li></ul>
+Week 5:
+<ul><li>Send push notifications to the Radio owners with less than a High potential signal level in the hex where they are located.</li></ul>
```
Comment (vincenzospaghetti) [inline 0074:162]: Through what? The black wallet app? What are you sending the notification through?

File: 0074 [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+5G Hotspot owners: 5G Hotspot owners with Outdoor Radios will need to meet new quality coverage requirements to earn maximum MOBILE rewards. Some owners might need to adjust the locations and/or angles of their Outdoor Radios to optimize for coverage. It is possible that in some places, it won't be possible to earn max rewards due to environmental obstacles that are not possible to mitigate. This HIP also proposes to consider rewarding those deployers who were among the first to provide quality coverage in a given Hex.
+
+There will be three possible outcomes for owners of Radios:
+<ol>
+<li>Increase in earnings. Because some Radios on the MOBILE Network won't be able to meet minimum quality requirements, part of the rewards will be re-distributed proportionally to those who qualify based on all PoC criteria.</li>
+<li>Earnings will stay the same.</li>
+<li>Less or no earnings.</li>
+</ol>
+
```
Comment (vincenzospaghetti) [inline 0074:38]: Does this impact manufacturers in any way? Do third-party mobile hotspot apps have to add anything to be compliant? Or get customers the information they need?

**Comment by vincenzospaghetti** [comment]
@meowshka - I've added comments here, can you make sure to answer these questions?

File: 0074 (hunk 1 above)
Comment (meowshka) [reply 0074:1]: MOBILE PoC - Modeled Coverage Rewards

File: 0074 (hunk 3 above)
Comment (meowshka) [reply 0074:139]: This will be a another web app, not related to the IoT Explorer.

File: 0074 (hunk 7 above)
Comment (meowshka) [reply 0074:38]: No, proposed changes do not impact manufacturers, and no action is needed on the third-party mobile apps.

File: 0074 (hunk 5 above)
Comment (meowshka) [reply 0074:153]: It will be available in the new Explorer dedicated for the MOBILE SubDAO.

File: 0074 (hunk 4 above)
Comment (meowshka) [reply 0074:152]: The plan is to allow users to locate their Radios in the new Mobile Explorer and see the modeled coverage from their Radio as an overlay on the map. The will be able to see how many hexed they cover, what are other Radios in that hex, and using the reference table, estimate the projected signal level and corresponding rewards.

File: 0074 (hunk 2 above)
Comment (meowshka) [reply 0074:51]: It's in the second paragraph, let me know if it needs more explanation. 

This HIP introduces a fundamental difference in the use of hexes for Mobile Network compared to IoT. In IoT, hexes are used to determine the locations of the Hotspots and the number of Hotspots in the given hex. On the contrary, in Mobile Network, hexes are used to determine where the coverage exists. The Radio doesn't have to be in a given hex to provide coverage there. The use of Modeled Coverage data allows us to un-tie the device's location from its signal strength and propagation. To sum up, the hexes in Mobile Network are used to identify the level of coverage in a particular hex from all the different radios. A Radio can earn rewards by providing coverage in multiple hexes.

**Comment by meowshka** [comment]
@vincenzospaghetti can you please add images? 
0074_hex_res_image1
0074_radio_installed_image2
0074_coverage_legend_image3

**Comment by meowshka** [comment]
@vincenzospaghetti I think table markup is off, can you please help with it? Here's how they are supposed to look 

Screen Shot 2022-11-29 at 1 40 20 PM
Screen Shot 2022-11-29 at 1 40 25 PM
Screen Shot 2022-11-29 at 1 40 31 PM
Screen Shot 2022-11-29 at 1 40 36 PM

**Pull request merged**